x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/string/InsertFunctionPipe.java

```diff
@@ -90,7 +90,7 @@ public InsertFunctionProcessor asProcessor() {
         return new InsertFunctionProcessor(source.asProcessor(), start.asProcessor(), length.asProcessor(), replacement.asProcessor());
     }
     
-    public Pipe source() {
+    public Pipe sourcePipe() {
```

Comment:
Maybe change the name of the rest of the methods to the `*Pipe()` format? It doesn't look consistent... `source` has its method called `sourcePipe()` while `start` and the rest have their methods called `start()`, `length()`, `replacement()`. Or maybe change the name of the `source` field to `sourceText` and have its return method called `sourceText()`. I would be in favor of the second option, to remain consistent with the rest of the functions (that don't have `Pipe` in their return methods).